File: wastage-reduction-final-master/feedol-ui/src/app/restaurant-profile.ts
export class RestaurantProfile{

    username:string;
    email:string;
    role:string;
    name:string;
    mobile:string;
    address:string;
    location:string;
    certificateNo:string;
    certificateName:string

    constructor(username, email, role, name, mobile, address, location, certificateNo,certificateName){
        this.username = username;
        this.email = email;
        this.role = role;
        this.name = name;
        this.mobile = mobile;
        this.address = address;
        this.location = location;
        this.certificateNo = certificateNo;
        this.certificateName = certificateName;
    }
}

File: wastage-reduction-final-master/registration-service/src/main/java/com/stackroute/registrationserver/domain/RestaurantProfile.java
package com.stackroute.registrationserver.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;



@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class RestaurantProfile {

    @Id
    private String username;
    private String email;
    private String role;
    private String name;
    private long mobile;
    private String address;
    private String location;
    private String certificateNo;
    private String certificateName;

}
